systest/scripts/cleanup_failed_tlc.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -x
+
+# We only need to cleanup the session, we didn't use barbican-enabled TLC file
+tlc --session ${TEST_SESSION} --debug cleanup && exit -1
```

Comment:
Something seems strange with this:

http://www.tldp.org/LDP/abs/html/exitcodes.html